SPARCore/GADArticle.m

```diff
@@ -2,6 +2,48 @@
 
 @implementation GADArticle
 
++ (void) fetchArticlesWithPublication: (NSString *)publicationId
+                                        completionHandler:(void(^_Nonnull)(NSArray<GADRemoteModel *>
+                                                                           *_Nullable models,
+                                                            NSError *_Nullable error))completion {
+    //Should we create NSDictionary parameters within each method and just request the publicationId
+    //from the front end?
+
+}
+
+- (void) populateArticleWithId: (NSString *)articleId
+                                        completionHandler:(void(^_Nonnull)(GADRemoteModel *
+                                                                           *_Nullable models,
+                                                            NSError *_Nullable error))completion {
+    //Similarly, should we create NSDictionary parameters within each method and just request the articleId
+    //from the front end?
+    
+}
+
++ (NSArray <GADArticle *> *) articlesFromJSON: (NSData *)json {
+    
+    NSMutableArray <GADArticle *> *articles = [[NSMutableArray alloc] init];
+    NSError *error = nil;
+    NSArray *jsonArray = [NSJSONSerialization JSONObjectWithData:json options:kNilOptions error:&error];
+    
+    if (error != nil) {
+        NSLog(@"Error parsing JSON.");
+    }
+    
+    for (NSDictionary *element in jsonArray) {
+        GADArticle *article = [[GADArticle alloc] init];
+        //Map fields of element to fields of article
+        article.datePublished = element[@"datePublished"];
```

Comment:
Yes, `static const string DATE_PUBLISHED = @"datePublished";` is perfect. If we wanted to get a little fancier, we could even add a relevant prefix like `API_`, thus: `static const string API_DATE_PUBLISHED = @"datePublished";`

Make sure they're in alphabetical order. :D